### ISSUE TITLE
Pattern: Content group - with pictograms - Accessibility - Screen reader is not able to identify pictogram icons, hence the user does not know the presence of these icons.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#1370]
## Detailed description
Devices : "JAWS FF, VO iPhone Safari"

Issue:- Screen reader is not identifying pictogram icons. hence the user does not know the presence of these icons.
Suggestion :- 
1. informative graphics should be given a descriptive alternative text so that the users of assistive technology can understand their purpose.
2. decorative/non informative graphics should be hidden from assistive technology so that they do not distract the user.

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/39487dc7-4e47-42df-941a-d8d2206c23be)

**Comment by RobertaJHahn** [comment]
@photodow James, this is the issue I mentioned to you.  The functional specs have to be updated to include the text descriptions of the pictograms before the developers can update the code.

**Comment by photodow** [comment]
Created https://github.com/carbon-design-system/ibm-dotcom-library/issues/1370 to track the functional spec work.

**Comment by RaphaelAmadeu** [comment]
We added the possibility of using aria attributes in order to improve the accessibility

**Comment by Praveen-sr** [comment]
@RaphaelAmadeu  We tested this on staging (https://ibmdotcom-react-experimental.mybluemix.net/) but issue still persists. Please let us know if its deployed on staging.

**Comment by RaphaelAmadeu** [comment]
@Praveen-sr The aria-description is being applied and it's working using chrome vox, what action do you recommend?

**Comment by RobertaJHahn** [comment]
This issue still persists.  Since the aria labels are applied Rahm will test to see whether this is a Jaws and FF bug.

**Comment by Praveen-sr** [comment]
@RobertaJHahn  Ram checked this issue and confirmed its a JAWS bug, we can close this one.

As of jaws version 2019 and 2020 in FF 73, JAWS is not identifying the role and the accessible name for SVG image in arrow key and quick navigation Key "G". Bug has been filed on the same in jaws bug tracker. below is the link.
https://github.com/FreedomScientific/VFO-standards-support/issues/373

**Comment by RobertaJHahn** [comment]
Closing this issue based on Praveeen's comments above.
